specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -54,6 +66,19 @@ clear-output-folder: true
 openapi-type: data-plane
 ```
 
+### Release 2022-05-01
+
+These settings apply only when `--tag=release_2022_05_01` is specified on the command line.
+
+``` yaml $(tag) == 'release_2022_05_01'
+input-file:
+  - stable/2022-05-01/textanalytics.json
+  - stable/2022-05-01/common.json
```

Comment:
Nit: you don't need to list common.json. It gets pulled in by tools automatically.

Reply:
removed  - stable/2022-05-01/common.json